chromeExtension/tests/integration/features/step-definitions/steps.js

```diff
@@ -1,450 +1,447 @@
-        Panes.Utilities.unlockAllFields();
-    }
-    else if (action === 'show-all-fields') {
-        Panes.Utilities.showAllFields();
-    }
-    else if (action === 'show-dirty-fields') {
-        Panes.Utilities.showDirtyFields();
-    }
-    else {
-        throw new Error('Unsupported action: ' + action);
-    }
-});
-
-When(/i disable a field/i, () => {
-    Panes.Utilities.disableField(RECORD_ATTRIBUTE);
-});
-
-When(/i hide a field/i, () => {
-    Panes.Utilities.hideField(RECORD_ATTRIBUTE);
-});
-
-When(/i change a field/i, () => {
-    Panes.Utilities.changeField(RECORD_ATTRIBUTE);
-});
-
-Then(/i should see the correct version number/i, () => {
-    expect(Panes.Toolbox.Version).toHaveText('v' + Package.version);
-});
-
-Then(/the toolbox display state should be (\w+)/i, state => {
-    if (state === 'collapsed') {
-        browser.waitUntil(() => Panes.Toolbox.ToolboxContainer.getSize('height') <= 100);
-        expect(Panes.Toolbox.ToggleButton).toHaveElementClass(state);
-    }
-    else if (state === 'expanded') {
-        browser.waitUntil(() => Panes.Toolbox.ToolboxContainer.getSize('height') > 100);
-        expect(Panes.Toolbox.ToggleButton).not.toHaveElementClass('collapsed');
-    }
-    else if (state === 'closed') {
-        expect(Panes.Toolbox.ToolboxContainer).not.toExist();
-    }
-});
-
-Then(/the toolbox should change position/i, () => {
-    expect(Panes.Toolbox.ToolboxContainer.getCSSProperty('top').parsed.value).toBeGreaterThanOrEqual(100);
-});
-
-Then(/i should see a value for these fields/i, (table) => {
-    table.rawTable.forEach(fields => {
-        const field = fields[0];
-
-        const valueElement = $(`[data-testid="${field}"] span`);
-        expect(valueElement).toExist();
-
-        const text = valueElement.getText();
-        expect(text).not.toBeNull();
-        expect(text.trim()).not.toBe('');
-    });
-});
-
-Then(/i should not see a value for these fields/i, (table) => {
-    table.rawTable.forEach(fields => {
-        const field = fields[0];
-
-        const valueElement = $(`[data-testid="${field}"] span`);
-        expect(valueElement).not.toExist();
-    });
-});
-
-Then(/i should see each field display the correct value/i, (table) => {
-    table.rawTable.forEach(fields => {
-        const field = fields[0];
-
-        const valueElement = $(`[data-testid="${field}"] span`);
-        expect(valueElement).toExist();
-
-        const text = valueElement.getText()?.toLowerCase();
-
-        switch (field) {
-            case 'record-id':
-                expect(text).toMatch(RECORD_ID);
-                break;
-            case 'logical-name':
-                expect(text).toMatch(RECORD_LOGICAL_NAME);
-                break;
-        }
-    });
-});
-
-Then(/i should see each field display a link to the correct entity/i, (table) => {
-    table.hashes().forEach(({ field, entity }) => {
-        const valueElement = $(`[data-testid="${field}"] span`);
```

Comment:
Killed this check off, I think it was actually a false positive success on the first key in the table b/c I refactored how this is rendered but it still took some time to load.